src/gui/mainwindow.cpp

```diff
@@ -1,328 +1,323 @@
 #include "mainwindow.h"
 
 #include <QCloseEvent>
```

Comment:
It seems a lot of this file was reformatted. Can we revert this change and keep just the important bits. It makes this a lot harder to read.

Reply:
Yeah I'll rever as much as I can